prprocessor/__main__.py

```diff
@@ -293,7 +304,7 @@ async def run_pull_request_check(pull_request: Mapping, check_run=None) -> bool:
         },
     )
 
-    return conclusion == 'success'
+    return (conclusion == 'success', issue_results, invalid_commits)
```

Comment:
no consumer uses `invalid_commits`, so we could also drop it from the return, but I thought maybe it's useful at some point later